### ISSUE TITLE
Weird comportement in android 2.2

### ISSUE THREAD

**Issue by ToYonos** [johannilsson/android-pulltorefresh#41]
The list comports itself weirdly the first time it is loaded.
Without scrolling, I tapped anywhere in the list and some elements kinda dissapeared and reappeared, so do the header. I scrolled a little the list and everything went back to normal.

How I fix it ? It took me hours but I have finally sorted it out : I removed the onAttachedToWindow method in the PullToRefreshListView class.

Is it the same in other android version ? I don't know, I haven't push my investigation any further.

**Comment by johannilsson** [comment]
Hi, thank you for the report. Sorry for being lazy here but can this be reproduced in the example app as well?

You could also try the [scrollfix_for_short_list](https://github.com/johannilsson/android-pulltorefresh/tree/scrollfix_for_short_list) which initializes itself a little bit different.

**Comment by ToYonos** [comment]
Yes, I tried the example app, without any modification, and the bug still appeared.

Actually, I don't really know if it's linked to the android version. Anyway, this trick works for me :)

**Comment by johannilsson** [comment]
Thank you for testing, will have a look at it and see if we can remove `onAttachedToWindow`.

**Comment by ToYonos** [comment]
And thank you for the lib

**Comment by since2006** [comment]
try to add a line in onAttachedToWindow:

super.onAttachedToWindow();

it's work fine to me with 2.2. :-)

**Comment by johannilsson** [comment]
@since2006 you're right that should be there, good catch.

**Comment by ToYonos** [comment]
@since2006 I tried this already and it didn't work at all.

Removing the overrided method was the only solution for me. (onAttachedToWindow() is still called but from the ListView class)
